docs/spec/slashing/README.md

```diff
@@ -0,0 +1,29 @@
+# Slashing module specification
+
+## Abstract
+
+This section specifies the slashing module of the Cosmos SDK, which implements functionality
+first outlined in the [Cosmos Whitepaper](https://cosmos.network/about/whitepaper) in June 2016.
+
+The slashing module enables Cosmos SDK-based blockchains to disincentivize any attributable action
+by a protocol-recognized actor with value at stake by "slashing" them: burning some amount of their
+stake - and possibly also removing their ability to vote on future blocks for a period of time.
```

Comment:
could be more clear to include these two items as numbered bullets - also use of "possibly" seems a bit out of place - I think slashing and jailing are independant penalties which are both _optional_ to a penalty